src/website/src/pages/ec/content-types/events/index.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: Events
+---
+
+import { Paragraph } from '@ecl/website-components';
+
+<Paragraph size="lead">
+  The event content type is organised and structured throughout multiple pages (
+  <a href="/ec/content-types/events/agenda/">agenda</a>,{' '}
```

Comment:
Native links are not wrong, but reload the page on browsing. To make the transition smoother, a `Link` component is provided. You can take it by `import { Paragraph, Link } from '@ecl/website-components'`. 

An example usage then will be 

```js
<Link to="/ec/components/forms/radio/usage/">radio buttons</Link>
```

Reply:
@kalinchernev Thank you, but I think there is an issue using Link inside a Paragraph, It render correctly but throwing an error during push:
`[error] src/website/src/pages/ec/content-types/events/index.mdx: SyntaxError: Void elements do not have end tags "Link" (3:53)`

I think this was the same for src/website/src/pages/ec/content-types/policy/index.mdx

Reply:
It's possible to include components inside by wrapping them in `{}`:

```js
---
title: Events
---

import { Paragraph, Link } from '@ecl/website-components';

<Paragraph size="lead">
  The event content type is organised and structured throughout multiple pages (
  {<Link to="/ec/content-types/events/agenda/">agenda</Link>},{' '}
  {<Link to="/ec/content-types/events/detail/">details</Link>},{' '}
  {<Link to="/ec/content-types/events/speaker/">speaker</Link>}) in order to minimise
  the overload of information.
</Paragraph>
<Paragraph size="lead">
  The content patterns for each page were created according to user research,
  which indicated how content of interest can be either{' '}
  <strong>arranged or progressively disclosed</strong>, by{' '}
  <strong>encouraging further navigation to the auxiliary pages</strong>.
</Paragraph>
```